cmd/drone-ecr/main.go

```diff
@@ -95,6 +122,30 @@ func ensureRepoExists(svc *ecr.ECR, name string) (err error) {
 	return
 }
 
+func uploadLifeCyclePolicy(svc *ecr.ECR, lifecyclePolicy string, name string) (err error) {
+	input := &ecr.PutLifecyclePolicyInput{}
+	input.SetLifecyclePolicyText(lifecyclePolicy)
+	input.SetRepositoryName(name)
+	_, err = svc.PutLifecyclePolicy(input)
+	if err != nil {
```

Comment:
You can skip that and always return err

Reply:
Ah, this makes much more sense to have handled with the if statement